coveralls.yml

```diff
@@ -1 +1 @@
-repo_token: COVERALLS_REPO_TOKEN
+repo_token: 7I4YACc9Iv2ByM57cGAZ8iKdKNLul3bPA
```

Comment:
Exposing this API key isn't a big deal, but you probably don't want it to be public knowledge. 

You **do not** want to expose your Amazon API key since it will be tied to someone's credit card. Be careful with sensitive data.

http://stackoverflow.com/a/2047477/2675670